README.md

```diff
@@ -0,0 +1,25 @@
+# terra-axon-examples
+
+## Overview
+
+This repository contains example notebooks and documentation for working with [Enterprise Terra](https://verily.com/solutions/terra/).
+
+[Enterprise Terra](https://verily.com/solutions/terra/) is a new platform developed by Verily to connect the multimodal data ecosystem and meet the needs of commercial or industry-led research organizations. Building on Terra’s Academic roots, Enterprise Terra is an integrated platform that provides the secure foundation to discover, govern, and unify multimodal data into a collaborative environment to accelerate discoveries.
+
+- Explore: discover multimodal data sets available in Enterprise Terra’s data ecosystem.
+- Unify: connect all data types, from both internal and external sources, into a single platform and under one governance framework.
+- Analyze: unlock insights faster with familiar and proprietary analysis tools.
+- Collaborate: share best practices and publish reproducible research.
+- Scale: maximize the power of any cloud with security, compliance, and support from industry standards.
+
+
+## Getting started
+
+### Setup Github integration in Enterprise Terra
+
+Go to [Enterprise Terra](https://verily.com/solutions/terra/) and:
+1. Add your SSH key to GitHub. 
+2. Create a personal workspace for yourself where you can test out many examples, such as this one. Name it something like *"Personal workspace {your ldap} {YYYYMMDD}"*. (You don't need to have multiple workspaces when you are just trying stuff out on public data, but if you want multiple workspaces, of course feel free to create them!)
+3. [Optional] If you want Terra to clone this repo automagically, add it as a reference in your workspace. Otherwise you will need to clone it manually, which is pretty easy.
```

Comment:
Do we expect users to know how to add references at this point?  Also, s/automagically/automatically/.

Reply:
Maybe not - when the docs become public I think it'd be good to include a link here to the section on references.